py/torch_tensorrt/_TRTModuleNext.py

```diff
@@ -0,0 +1,202 @@
+import logging
+from operator import truediv
+from typing import Any, List, Sequence, Tuple
+
+import torch
+from torch_tensorrt import _C
+from torch_tensorrt._Device import Device
+
+logger = logging.getLogger(__name__)
+
+
+class TRTModuleNext(torch.nn.Module):
```

Comment:
Consider adding a `TRTModuleNext.close()` function, as performing inference with the module and then closing the python session results in an error:
```python
ERROR: [Torch-TensorRT - Debug Build] - 3: [engine.cpp::~Engine::305]
Error Code 3: API Usage Error (Parameter check failed at: runtime/api/engine.cpp::~Engine::305,
condition: objectCount() == 0. Destroying an engine object before destroying objects it created
leads to undefined behavior.)
```